### ISSUE TITLE
[feature request] Upgrade Quasar libraries

### ISSUE THREAD

**Issue by bingalls** [pratik227/quasar-admin#4]
Quasar-admin is breath-taking!
Please upgrade Quasar-admin's dependent (Quasar) libraries.
tldr;
I am building an example open app on Github, to kickstart and show off a full-stack app using quasar-admin.
This is using latest in best-of-breed, including GraphQL & Docker Compose.
I'm inexplicably running into difficulty getting @quasar/app to install.
Essentially, Docker installs @quasar/cli but not @quasar/app.
I decided to remedy this, by upgrading the libraries that Quasar-admin uses.
Following the [Quasar Upgrade Guide](https://quasar.dev/quasar-cli/app-upgrade-guide#Upgrade-Guide) I ran into difficulties which I believe is in the viewport meta tag in index.template.html, which I had to modify as part of the upgrade.
I'll be happy to post my evolving code, should you wish to see details, or test it, youself.
FWIW, I can attest that quasar-admin works on node v14 lts & node v15, as well as with Yarn.
Thanks!

**Comment by pratik227** [comment]
Hi,

Yes, you can post code here I will check and merge it quasar-admin.

Thanks!!

**Comment by patrickmonteiro** [comment]
I migrated to the latest version of quasar.
Just wait to be tested.

https://github.com/pratik227/quasar-admin/pull/5

**Comment by pratik227** [comment]
@bingalls Upgrade Quasar libraries

**Comment by bingalls** [comment]
@pratik227 Sorry about delay in reply. Extracting my code for this was taking time.
Upgrade looks good! I recommend documenting a release procedure, so that you remember to update the version in package.json. Here are issues I found, while testing in Firefox Developer Edition v85 (latest)
* Open http://localhost:8080/Mail in a new tab. In mobile mode, the (hamburger) menu does not change, so you can't understand why you can't return to the main menu.
* http://localhost:8080/Charts warns of 
`Invalid prop: type check failed for prop "option". Expected Object, got Null 
found in
---> <IEcharts>`
* http://localhost:8080/Pricing lost the hamburger menu, although a new dashboard icon helps.

**Comment by pratik227** [comment]
@bingalls I have fixed the issues.

- [✓] Open http://localhost:8080/Mail in a new tab. In mobile mode, the (hamburger) menu does not change, so you can't understand why you can't return to the main menu.
- [✓] http://localhost:8080/Charts warns of Invalid prop: type check failed for prop "option". Expected Object, got Null found in ---> <IEcharts>

For the Pricing page I have kept the dashboard icon so user can move to the dashboard. The loss of the hamburger menu is not the issue.
